lib/BridgedRoom.js

```diff
@@ -300,4 +391,68 @@ BridgedRoom.prototype.getMatrixATime = function() {
     return this._matrixAtime;
 };
 
+BridgedRoom.prototype.lookupAndSetTeamInfo = function() {
+    if (!this._slack_bot_token) return Promise.resolve();
+
+    return rp({
+        uri: 'https://slack.com/api/team.info',
+        qs: {
+            token: this._slack_bot_token
+        },
+        json: true,
+    }).then((response) => {
+        if (!response.team) return;
```

Comment:
Why not `Promise.resolve()`?

Reply:
there's no need, since we're already in a promise

Reply:
Ok, but your are doing it line 447.

Reply:
oops. removed.